all_test.go

```diff
@@ -38,8 +37,11 @@ func TestCopy(t *testing.T) {
 	})
 
 	When(t, "specified src is just a file", func(t *testing.T) {
-		err := Copy("test/data/case01/README.md", "test/data.copy/case01/README.md")
+		err := Copy("test/data/case00/README.md", "test/data.copy/case00/README.md")
```

Comment:
Why did you change case01 to case00 specifically here?

Reply:
I added some content to the case00/README.md and check then the content of the file as well. I can revert this also if you want.

Reply:
This is a test case for "the src is just a file", and I use `case01` for this case. So if you think this case should contain some contents, you can add some text into `case01`.

Reply:
Oh I see the point of the `casexx` files now. I reverted this.